src/TensorKit.jl

```diff
@@ -220,6 +233,8 @@ include("auxiliary/deprecate.jl")
 # ----------
 function __init__()
     @require_extensions
+    set_num_transformer_threads(Threads.nthreads())
```

Comment:
Is that the default you wanted? Or did you want to just use a single thread (and only BLAS threads). Or is that equivalent because you don't run julia with multiple threadsd?

Reply:
Good point, probably better to leave this off by default.